doc/WebRTC.xml

```diff
@@ -1,1029 +1,653 @@
+            <para role="param">410 Gone</para>
+            <para role="text">The peer is no more available.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+    </section>
+    <section>
+      <title xml:id="section_setIceCandidate">trickle</title>
+      <para>An ONVIF compliant signaling server, device and client shall support this command to
+        signal new ICE candidates for the trickleICE procedure as defined in RFC 8840.</para>
+      <para>A signaling server shall relay this command unaltered to the peer.</para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>request</term>
+          <listitem>
+            <para role="param">session [string]</para>
+            <para role="text">The ID assigned by the signaling server to the session. </para>
+            <para role="param">candidate [IceCandidate]</para>
+            <para role="text">The ICE Candidate SDP update for the peer. </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>response</term>
+          <listitem>
+            <para role="text">&lt;none&gt;</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>faults</term>
+          <listitem>
+            <para role="text">&lt;none&gt;</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+      <para>Both client and device produce ICE candidates and send them to each other, via the
+        server, in any order.</para>
+      <para><emphasis role="bold">NOTE</emphasis>: Note that ICE candidates can arrive <emphasis
+        role="bold">before</emphasis> the SDP offer and the implementing client needs to handle
+        this.</para>
+      <para>If everything works as it should, a peer-to-peer WebRTC session can be set up between
+        client and device. After the session has been established the client can terminate the
+        WebSocket session to the signaling server and the peer-to-peer connection will not be
+        affected. However, it's recommended to keep the WebSocket session to the server open to be
+        able to receive event messages from the signaling server.</para>
+    </section>
+    <section>
+      <title>Example Flow (informative)</title>
+      <para>The following example shows the flow using JSON-RPC 2.0. Note that for improved
+        readability the madnatory JSON version string is not shown.</para>
+      <programlisting><![CDATA[
+Client -> Server: { "method": "register", "params": {"authorization": "JWT..."}, "id":1}
+Server -> Client: { "result": { "id": "client-a1" }, "id": 1}
+Device -> Server: { "method": "register", "params": {"authorization": "JWT..."}, "id":1}
+Server -> Device: { "result": { "id": "device-b1" }, "id": 1}
+
+Client -> Server: { "method": "connect", 
+              "params": {"authorization": "JWT...", "DeviceID": "device-b1"}, "id":2}
+Server -> Device: { "method": "connect", 

```

Comment:
Have taken the semantics from the original proposal. Understand the rationale for passing the token with connect in order to ensure that a fresh token is passed. As the time between connect, invite and trickle is short I don't see a reason to resent the token. As all communication is happening over a TLS connection there should be no need to protect individual messages.

Reply:
I believe there could be some scenarios, like renegotiation after network change, which could happen after longer period of time, but since we aren't supporting it, its probably not an issue.